pkg/web/oAuthHandler.go

```diff
@@ -8,26 +8,53 @@ import (
 	"net/http"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/feedhenry/mcp-standalone/pkg/k8s"
+	"github.com/feedhenry/mcp-standalone/pkg/mobile"
 	"golang.org/x/oauth2"
 )
 
 // OAuthHandler handle oauth actions
 type OAuthHandler struct {
-	logger *logrus.Logger
-	config *oauth2.Config
+	logger        *logrus.Logger
+	k8sMetadata   k8s.Metadata
```

Comment:
possible to group into a struct

Reply:
not going to do this after trying.
One field is a derived field, and not part of the params, so there would only be 3 things in the struct.
The constructor `NewOauthHandler` would still need to take 3 params instead of 4, so not enough saving to combine into a struct at this time